feature/amazon/src/main/java/com/revenuecat/purchases/amazon/productConverter.kt

```diff
@@ -0,0 +1,64 @@
+package com.revenuecat.purchases.amazon
+
+import com.amazon.device.iap.internal.model.ProductBuilder
+import com.amazon.device.iap.model.Product
+import com.revenuecat.purchases.models.ProductDetails
+import org.json.JSONObject
+import java.text.NumberFormat
+import java.text.ParseException
+import java.util.Currency
+import java.util.Locale
+import com.amazon.device.iap.model.ProductType as AmazonProductType
+
+val ProductDetails.amazonProduct: Product
+    get() = ProductBuilder()
+        .setSku(originalJson.getString("sku"))
+        .setProductType(originalJson.getProductType("productType"))
+        .setDescription(originalJson.getString("description"))
+        .setPrice(originalJson.getString("price"))
+        .setSmallIconUrl(originalJson.getString("smallIconUrl"))
+        .setTitle(originalJson.getString("title"))
+        .setCoinsRewardAmount(originalJson.getInt("coinsRewardAmount")).build()
+
+fun Product.toProductDetails(marketplace: String): ProductDetails {
+    val numberFormat = NumberFormat.getInstance()
+
+    val currency = Currency.getInstance(Locale("EN", marketplace))
+    val currencySymbol = currency.symbol
+
+    val priceDouble: Double =
+        if (price.startsWith(currencySymbol)) {
+            // Looks like Amazon always prefixes the price with the currency, no matter the Locale
+            val formattedPriceWithoutSymbol = price.removePrefix(currencySymbol)
```

Comment:
would it be safer to find the range of the string and remove it? or to try prefix and then if it wasn't removed, try suffix? 
it looks like this should be safe enough from your tests, but it might not hurt

Reply:
I like the idea. I bet there will be a strange case that I haven't tested we don't cover, but let's try to cover most cases.